src/lessons/02_introduction_to_frontend/introduction-to-frontend.slides.html

```diff
@@ -0,0 +1,729 @@
+              <li>Why knowing <b>CSS</b></li>
+              <li>
+                <a href="https://www.w3.org/standards/webdesign/htmlcss.html" target="_blank">W3C</a> standards
+              </li>
+            </ul>
+          </section>
+        </section>
+
+        <!--SLIDE 13-->
+        <!--Inline CSS (examples)-->
+        <section>
+          <!--SLIDE 13.1-->
+          <section data-auto-animate data-timing="10">
+            <h3>For now, let's use "inline" CSS</h3>
+          </section>
+
+          <!--SLIDE 13.2-->
+          <section data-auto-animate data-timing="10">
+            <h3>For now, let's use "inline" CSS</h3>
+            <img data-src="images/wat.jpg" class="r-frame" />
+          </section>
+
+          <!--SLIDE 13.3-->
+          <section data-auto-animate data-timing="300">
+            <h3>For now, let's use "inline" CSS</h3>
+            <p class="small-please">You most likely remember our tag attributes from some of the tags, don't ya?</p>
+
+            <pre data-auto-animate><code data-trim class="html"><script type="text/template">
+              <html>
+                <head><title>Inline CSS</title></head>
+                <body>
+                  <p>This is unstyled text</p>
+                  <p style="color: blue">This text is blue</p>
+                  <p style="color: red; background-color: yellow">
+                    Multiple properties can be separated by semicolons
+                  </p>
+                </body>
+              </html>
+            </script></code></pre>
+
+            <aside class="notes">
+              <li>First, add this attribute for any structural element of your page: <code>style=""</code></li>
+              <li>As an example, insert a color for this element by adding this "color" property: <code style="font-size: .5em">style="color: blue;"</code></li>
+              <li>Remember: CSS contains properties which can define different styles, colors, margins, etc; for your element</li>
+              <li>Remember: For now, as we're using inline, let's make sure it's written as <code style="font-size: .5em">style="PROPERTY: VALUE; PROPERTY2: VALUE2;"</code></li>
+              <li>Remember: We want to separate HTML from CSS (beware we're deep diving on this in the next lesson!</li>
+            </aside>
+          </section>
+
+          <!--SLIDE 13.4-->
+          <section data-auto-animate data-timing="10">
+            <h3>For now, let's use "inline" CSS</h3>
+
+            <div style="text-align: left" class="r-fit-text">
+              <p class="small-please"><b>Q</b>: Still, why "inline" and what does that mean❓</p>
+              <p class="small-please current-value"><b>A</b>: It means we want to make sure we add styles for a few components to get an idea of how CSS works</p>
+            </div>
+
+            <p class="fragment small-please">👉 We will dive deeper into CSS in the next lesson.</p>
```

Comment:
Now that this text is on a separate slide, the text can be a bit larger in my opinion. Otherwise the slide feels so empty 😄

Reply:
agreed 😛

Reply:
Same, the rest is fine by me :)